packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,322 @@
+import * as React from 'react';
+import styled from '@emotion/styled';
+import { CreateStyled } from '@emotion/styled';
+
+import { ThemeContext } from '../../theme/ThemeContext';
+import { ThemeInterface } from '../../theme/magma';
+import { useIsInverse } from '../../inverse';
+import { Step, StepProps, StepStatus } from './Step';
+import { transparentize } from 'polished';
+import { I18nContext } from '../../i18n';
+
+export interface StepperProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * @internal
+   */
+  testId?: string;
+  /**
+   * Custom number for responsive styling, uses a minimum 'breakpoint' width from the set number.
+   */
+  breakpoint?: number;
+  /**
+   * Changes the Stepper view for responsive layouts, needs a minimum 'breakpoint' number.
+   */
+  breakpointLayout?: StepperLayout;
+  /**
+   * Sets the Stepper view
+   * @default StepperLayout.showLabels
+   */
+  layout?: StepperLayout;
+  /**
+   * Sets a custom label for the Step count # of #.
+   * @default 'Step'
+   */
+  stepLabel?: string;
+  /**
+   * Sets a custom label for the completed Step.
+   * @default 'All steps completed'
+   */
+  completionLabel?: string;
+  /**
+   * Current step value.
+   */
+  currentStep?: number;
+  /**
+   * Inverse styling.
+   */
+  isInverse?: boolean;
+  /**
+   * @internal
+   */
+  theme?: ThemeInterface;
+}
+
+export enum StepperLayout {
+  showLabels = 'showLabels', //default
+  hideLabels = 'hideLabels',
+  summaryView = 'summaryView',
+}
+
+function buildSeparatorBackgroundColors(props) {
+  const { isInverse, theme, stepStatus } = props;
+  if (isInverse) {
+    if (stepStatus === StepStatus.complete) {
+      return theme.colors.tertiary500;
+    }
+    return theme.colors.primary400;
+  } else {
+    if (stepStatus === StepStatus.complete) {
+      return theme.colors.primary500;
+    }
+    return theme.colors.neutral300;
+  }
+}
+
+const typedStyled = styled as CreateStyled<ThemeInterface>;
+
+const StyledStepper = typedStyled.div`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+`;
+
+const StyledStepContent = typedStyled.div<{
+  showLabelsLayout?: boolean;
+  theme?: ThemeInterface;
+}>`
+  display: flex;
+  
+`;
+
+const StyledWrapper = typedStyled.div`
+    position:relative;
+    flex:1;
+`;
+
+const StyledSeparator = typedStyled.div<{
+  isInverse?: boolean;
+  showLabelsLayout?: boolean;
+  stepStatus: StepStatus;
+  theme?: ThemeInterface;
+}>`
+  background: ${buildSeparatorBackgroundColors};
+  width: ${props => (props.showLabelsLayout ? 'calc(100% - 24px)' : '100%')};
+  height: 2px;
+  top: 11px;
+  left: ${props => (props.showLabelsLayout ? 'calc(50% + 12px)' : '')};
+  position: ${props => (props.showLabelsLayout ? 'absolute' : 'relative')};
+  align-self: baseline;
+  transition: all 0.4s ease;
+`;
+
+const StyledSummary = typedStyled.div<StepperProps>`
```

Comment:
Did we lose some space between the stepper and the summary? Feels a little tight
<img width="147" alt="image" src="https://github.com/cengage/react-magma/assets/91160746/9a71589b-444f-40ec-9d48-86a59e3aa8f3">

Reply:
Agreed. If you put borders on the indicator portion, and the container for the labels, there should be 8px of space in between.
![image](https://github.com/cengage/react-magma/assets/24397282/128733e4-b486-4d71-a549-b676f173eac7)